packages/dd-trace/src/profiling/config.js

```diff
@@ -46,12 +48,13 @@ class Config {
     this.service = service
     this.env = env
     this.host = host
+    this.functionName = functionName
 
     this.version = version
     this.tags = Object.assign(
       tagger.parse(DD_TAGS),
       tagger.parse(options.tags),
-      tagger.parse({ env, host, service, version })
+      tagger.parse({ env, host, service, version, functionName })
```

Comment:
Is `functionName` the actual tag name? Usually the tag names are in the form `function-name` or `function_name`.

Reply:
It's actually `functionname` due to a mistake made years ago that we sort of have to live with

Reply:
@amymin00 Can you change this to `functionname` as mentioned above? Totally missed that this wasn't already done, but let's make sure we use the proper casing to avoid issues.